src/lua/share/scripts/geoip_demo.lua

```diff
@@ -0,0 +1,41 @@
+---
+
+description = [[ This demonstrates GeoIP information in ettercap.
+                 Ettercap MUST have been compiled '-DWITH_GEOIP'.
+              ]]
+
+local ffi         = require ('ettercap_ffi')
+local packet      = require ("packet")
+local hook_points = require ("hook_points")
+
+-- Hook at the filtering point
+--
+hook_point = hook_points.filter
+
+is_ip = function(p)
+  return (p.L3 and p.L3.proto == 0x8)
+end
+
+-- Optional, but we only want to look at IP-packets.
+--
+packetrule = function (p)
+  if not is_ip(p) then
```

Comment:
It should perhaps not ignore IPv6 packets. `p.L3.proto == 0xDD86`. Untested.

Reply:
Please test.

Reply:
Just change it into:
```
is_ip = function(p)
  return (p.L3 and (p.L3.proto == 0x8 or p.L3.proto == 0xDD86))
end
```
but I cannot verify this since I have no IPv6-connections to the world. I still use my  good old Linux based dd-Wrt router for now (and not the crap my ISP gave me).

Reply:
I also don't have IPv6-connections to the world.
This can work though...